ocs_ci/deployment/helpers/hypershift_base.py

```
@@ -1097,6 +1097,51 @@ def download_hosted_cluster_kubeconfig(
+        logger.info(
+            f"Downloading kubeadmin-password for HyperShift hosted cluster "
+            f"{name} to {password_path}"
+        )
+
+        try:
+            with config.RunWithProviderConfigContextIfAvailable():
+                OCP().exec_oc_cmd(
+                    f"extract secret/kubeadmin-password -n clusters-{name} "
+                    f"--to {auth_path} --confirm"
+                )
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)